src/cli/cli_coap.cpp

```diff
@@ -583,11 +583,12 @@ otError Coap::ProcessRequest(Arg aArgs[], otCoapCode aCoapCode)
     otError       error   = OT_ERROR_NONE;
     otMessage    *message = nullptr;
     otMessageInfo messageInfo;
-    uint16_t      payloadLength = 0;
+    uint16_t      payloadLength    = 0;
+    char         *uriQueryStartPtr = nullptr;
 
     // Default parameters
-    char         coapUri[kMaxUriLength] = "test";
-    otCoapType   coapType               = OT_COAP_TYPE_NON_CONFIRMABLE;
+    char         coapUri[kMaxUriLength]      = "test";
+    otCoapType   coapType                    = OT_COAP_TYPE_NON_CONFIRMABLE;
```

Comment:
To pass the pretty check:
```suggestion
    char         coapUri[kMaxUriLength] = "test";
    otCoapType   coapType               = OT_COAP_TYPE_NON_CONFIRMABLE;
```